### DISCUSSION TITLE
Welcome to SCTE35-threefive Discussions!

### DISCUSSION THREAD

**Discussion by futzu** [futzu/threefive#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by futzu** [comment]
### Let's have a discussion

**Comment by GBuchhardt** [comment]
Hello I'm using threefive and it works  fine. 
Now  I have an  new TS (only audio) that does not detected  the markers. 
Other DVB-Analyser work fine with that and detect it. Example: https://iptvweb.anixe.tv/public/scte35_out.ts.
Why does threefive fail here ? Is there a way to get the binary payload of the scte35 pid ?

**Reply by futzu** [reply]
Your web server is down and I can't get the video.
Are you using the latest version 2.4.51?

**Reply by futzu** [reply]
What's up man ? You need help or not? 
Your web server is still down.
I am a busy man.

**Reply by futzu** [reply]
I guess you solved the problem.

**Reply by GBuchhardt** [reply]
I answered you via PM, seems that you did not get all my mails.  Webserver link works but without https, certificate is broken, sorry,  so I used an other Webserver for sending the link. Finally a update to the latest version of threefive solved my problem,  sap_type is now handled correctly. Great software , thank you for providing it via github. Gerd

**Reply by futzu** [reply]
Cool, I just uploaded a new release 2.4.53,  it handles ffmpeg putting PES headers in SCTE-35 packets, which is kind of  difficult to spot.  Grab it just in case.

**Reply by futzu** [reply]
there is a bug in 2.4.53, please use 2.4.55
